### DISCUSSION TITLE
Homepage should allow faster search

### DISCUSSION THREAD

**Discussion by xawos** [community/community#120018] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

If you're not logged in you need to do two taps from the mobile web interface in order to access the search functionality.

Maybe I'm an edge case, but when I use the mobile webapp I only search repos, I'd love to have the search icon somewhere there without having to login.

Happy computing :)

**Comment by LiteBrite82** [comment]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
